### PR TITLE
[ROCm] Fixes to make //tensorflow/compiler/xla/service/gpu/tests:mlir_gemm_t…

### PR DESCRIPTION
**Pull request by rsanthanam-amd**
…est work with TFRT BEF Thunk on ROCm.

### CODE REVIEW

File: tensorflow/compiler/mlir/tfrt/transforms/lmhlo_to_gpu/gemm_pattern.cc [hunk 1]
```diff
@@ -44,9 +44,12 @@
 #include "tfrt/gpu/kernels/gpu_ops.h"  // from @tf_runtime
 #include "tfrt/gpu/passes/passes.h"  // from @tf_runtime
 #include "tfrt/gpu/wrapper/cublas_wrapper.h"  // from @tf_runtime
+#include "tfrt/gpu/wrapper/rocblas_wrapper.h"  // from @tf_runtime
 #include "tfrt/basic_kernels/opdefs/basic_kernels.h"  // from @tf_runtime
 #include "tfrt/basic_kernels/opdefs/types.h"  // from @tf_runtime
 
+#include <iostream>
```
Comment (chsigg) [inline tensorflow/compiler/mlir/tfrt/transforms/lmhlo_to_gpu/gemm_pattern.cc:51]: I think you can remove this againn?

Comment (rsanthanam-amd) [reply tensorflow/compiler/mlir/tfrt/transforms/lmhlo_to_gpu/gemm_pattern.cc]: fixed (sorry).

**Comment by rsanthanam-amd** [comment]
@chsigg I fixed the merge conflicts in this PR.

**Comment by chsigg** [comment]
Thanks Rohit, this looks great, thanks a lot! I will merge this change manually.

**Pull request merged**